modular_skyrat/modules/goofsec/code/projectiles.dm

```diff
@@ -201,3 +201,279 @@
 	damage = 60
 */
 
+/// Bullet security pistol.
+/obj/item/gun/ballistic/automatic/pistol/security
+	name = "security pistol"
+	desc = "A pistol issued to station security personnel. It's a small, compact pistol, but it packs a punch."
+	mag_type = /obj/item/ammo_box/magazine/security_pistol
+	icon = 'modular_skyrat/modules/sec_haul/icons/guns/ladon.dmi'
+	righthand_file = 'modular_skyrat/modules/sec_haul/icons/guns/inhands/righthand40x32.dmi'
+	lefthand_file = 'modular_skyrat/modules/sec_haul/icons/guns/inhands/lefthand40x32.dmi'
+	icon_state = "ladon"
+	inhand_icon_state = "ladon"
+	can_suppress = FALSE
+	fire_sound = 'modular_skyrat/modules/sec_haul/sound/pistol_fire.ogg'
+	rack_sound = 'sound/weapons/gun/pistol/rack.ogg'
+	lock_back_sound = 'sound/weapons/gun/pistol/slide_lock.ogg'
+	bolt_drop_sound = 'sound/weapons/gun/pistol/slide_drop.ogg'
+
+/obj/item/ammo_box/magazine/security_pistol
+	name = "influence security pistol magazine"
```

Comment:
Again with the names, make it obvious what the round does. Is it rubber? Then name it that.